geotrek/settings/base.py

```diff
@@ -33,7 +32,7 @@ def api_bbox(bbox, buffer):
 
 DOT_ENV_FILE = os.path.join(VAR_DIR, 'conf/env')
 if os.path.exists(DOT_ENV_FILE):
-    env_file.load(path=DOT_ENV_FILE)
+    load_dotenv(DOT_ENV_FILE)
```

Comment:
**Suggestion:** Consider using `pathlib` for file and directory path manipulations as it is more readable and less error-prone. [enhancement]
```suggestion
from pathlib import Path
DOT_ENV_FILE = Path(VAR_DIR) / 'conf' / 'env'
if DOT_ENV_FILE.exists():
    load_dotenv(DOT_ENV_FILE)
```